<?xml version="1.0"?>
<launch>

	<node name="prob_road_pub" pkg="dynamic_obstacle_remover" type="probability_road" output="screen">		
		
		<param name="Parent_id" value="/velodyne" />
		<param name="Child_id" value="/odom" />

		<param name="skip_time" value="5" />
		<param name="step_num" value="10" />
		<rosparam file="$(find intersection_intensity)/yaml/save.yaml"/>
		<!-- <param name="radius_length" value="20" /> -->
		<param name="per_cell_" value="0.2" />
		<param name="static_threshold" value="0.6" />
	</node>

</launch>



